lib/Cargo.toml

```diff
@@ -13,7 +13,7 @@ keywords = ["gis", "geo", "geography", "geospatial"]
 geo-types = { version = "0.4", default-features = false }
 num-traits = "0.2"
 robust = "0.1"
-float_next_after = "0.1.2"
+float_next_after = "0.1"
```

Comment:
Interesting, the newer version has a lower number. I though cargo makes sure that versions cannot be overwritten and can only "move forward".

Reply:
Yeah, I changed it from 0.1.2 to 0.1 since cargo believes in semantic versioning.  If you put 0.1.2, it will use 0.1.2, but if you use 0.1 it will pull the latest version in 0.1.x (which is currently 0.1.5).  

Probably I should have changed the version to 0.2 when we changed away from the reference interface.